src/flowchem/devices/vicivalco/vici_valve_component.py

```diff
@@ -15,7 +15,8 @@ class ViciInjectionValve(SixPortTwoPositionValve):
     def _change_connections(self, raw_position: int, reverse: bool = False) -> str:
         raise NotImplementedError("Check that provided mapping is correct")
         # TODO maybe needs addition of one, not sure
-        if not reverse:
+
+        if not reverse: # Is it right?
```

Comment:
it is right if position 0 and 1 in the valve internal command maps to the positions that the implemented stator and rotor layout of the parent class provided these connections in position 0 and 1

Reply:
pls remove the comment